sonar-application/src/main/java/org/sonar/application/App.java

```diff
@@ -171,15 +171,21 @@ public static void main(String[] args) throws InterruptedException {
     CommandLineParser cli = new CommandLineParser();
     Properties rawProperties = cli.parseArguments(args);
     Props props = new PropsBuilder(rawProperties, new JdbcSettings()).build();
+    ClusterProperties clusterProperties = new ClusterProperties(props);
+    clusterProperties.populateProps(props);
+
     AppFileSystem appFileSystem = new AppFileSystem(props);
     appFileSystem.verifyProps();
     AppLogging logging = new AppLogging();
     logging.configure(props);
+    clusterProperties.validate();
```

Comment:
it's strange to not have this line with the two others lines related to variable `clusterProperties`